Dockerfile.amd64.cross

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.10.5-stretch
```

Comment:
Actually, we all use golang 1.10.4. for building pouchd binaries.
Could we update that to be 1.10.4 to keep unified? @houstar